src/bindings/python/_flux/Makefile.am

```diff
@@ -31,7 +33,7 @@ _core_build.py: $(MAKE_BINDING)
 		--add_sub '.*va_list.*|||' \
 		--additional_headers src/bindings/python/_flux/callbacks.h \
 		--ignore_header 'macros' \
-		src/include/flux/core.h
+		src/include/flux/core.h src/include/flux/debugged.h
```

Comment:
Isn't `debugged.h` in `src/common/libdebugged/debugged.h`, not `src/include/flux/debugged.h`?

Reply:
`src/include/flux/debugged.h` is there as well, in which I included `src/common/libdebugged/debugged.h`. Maybe I didn't check that in? I will check and add that file if not committed in.

Reply:
Oh, is libdebugged/debugged.h part of the public API?

Reply:
> Oh, is libdebugged/debugged.h part of the public API?

Depends what you mean by this. But it needs that to make python binding code a bit simple by allowing. 

```python
   for header in absolute_heads:
        process_header(header)
        includes += "#include <{}>\n".format(header)
```

Also I was to try to link it with `flux-job` to see if that will disallow compilers to use optimization on MPIR_Breakpoint function.

Does it make it public enough?

Reply:
Hm, I'm probably not familiar enough with the python cffi stuff to understand how this works, but for an installed flux-core, how will the python `flux.debugged` module get the symbols from `libdebugged`?

Instead of being a cffi module could `flux.debugged` be a .so module compiled against libdebugged.la convenience library so that libdebugged.la and its symbols don' t need to be added to libflux-core.so?

In either case, I don't think is should be necessary to install `/usr/include/flux/debugged.h` if that isn't meant to be part of the libflux api (just part of the python api), so perhaps `src/include/debugged.h` wrapper isn't required?

Reply:
The symbols will be first included into `_core` and `flux.debugged` will get them from there. So the MPIR symbols will not be included into `libflux-core.so`. 

I actually agree this header file don't need to be installed in the public location. 

Maybe I should change `make_python.py` to use double quotes (`"`) instead of angle brackets (`'<'` etc) for `#include` used with `ffi.cdef()`. If this works, we should be good. I will test this but @SteVwonder or @trws will ultimately have to review it.

Reply:
Ah, ok that sounds perfect then!